server/src/entities/activity.entity.ts

```diff
@@ -40,10 +40,10 @@ export class ActivityEntity {
   @Column({ type: 'boolean', default: false })
   isLiked!: boolean;
 
-  @ManyToOne(() => AssetEntity, { onDelete: 'CASCADE', onUpdate: 'CASCADE', nullable: true })
+  @ManyToOne(() => AssetEntity, { onDelete: 'CASCADE', nullable: true })
   asset!: AssetEntity | null;
 
-  @ManyToOne(() => UserEntity, { onDelete: 'CASCADE', onUpdate: 'CASCADE' })
+  @ManyToOne(() => UserEntity, { onDelete: 'CASCADE' })
```

Comment:
This change seems unnecessary. Why did you make it?

Reply:
Because when an asset is trashed, that row becomes `NULL`, so you can't differentiate a global comment from an asset comment.

Reply:
But that has nothing to do with this cascade.